src/factory.css

```diff
@@ -607,3 +607,53 @@ td.taboff:hover {
 .shadowBlock>.blocklyPathDark {
   display: none;
 }
+
+ /* New block popup */
+ 
+#popup {
+  display: none;
+  height: 100%;
+  left: 0;
+  position: absolute;
+  top: 0;
+  width: 100%;
+  z-index: 99999999999999999999999999999;
```

Comment:
Needs to be high to be higher than the scroll bars.

A future design consideration (no need to address in this PR):
Why are we choosing a modal interface? Why are we not allowing the user to interact with the project tree or file export button?  Why are we loading a blockly workspace at all before the block is created (the source of your z-index problems).  Can we have this creation wizard interface loaded into the same <div> panel as all the other editors, before the block editor is loaded?